### ISSUE TITLE
Grafana Not Showing Any Data in Graphs from Zabbix.

### ISSUE THREAD

**Issue by baxtmann** [alexanderzobnin/grafana-zabbix#980]
**Describe the bug**
No data from zabbix is appearing in grafana. Hosts/groups are also not showing up when creating a graph. This most likely is a configuration issue with the data source, but as far as I can tell, it all should be working...

**Expected behavior**
Grafana should show data. 

**Screenshots**
![image](https://user-images.githubusercontent.com/18515115/83791099-418fb200-a667-11ea-8a06-e9bb62e4d506.png)
![image](https://user-images.githubusercontent.com/18515115/83791089-3d639480-a667-11ea-91e7-492cc9dcc391.png)
![image](https://user-images.githubusercontent.com/18515115/83791125-4bb1b080-a667-11ea-8254-3fa089ee1bc6.png)
As shown by the last photo, there should be a drop down menu by hosts and groups allowing you to select host/group, but there is not any. 

**Network data**
Just as a note: when I run a query with query inspection the query just spins and doesnt do anything. If I refresh the page I get the response below. not sure if its useful or not...
`{
  "request": {
    "method": "POST",
    "url": "api/datasources/proxy/2",
    "data": {
      "jsonrpc": "2.0",
      "method": "usermacro.get",
      "params": {
        "output": "extend",
        "hostids": []
      },
      "id": 1,
      "auth": "80eaed92f7d9106603b41f724b41f68d"
    }
  },
  "response": {
    "jsonrpc": "2.0",
    "result": [],
    "id": 1
  }
}`

**Software versions**

| Grafana | Zabbix | Grafana-Zabbix Plugin |
| ------- | ------ | --------------------- |
| 7.0.3   | 5.0.0  | 3.12.5       |

Basically I think the datasource config is wrong, but I cant figure out what the issue is. This is a fresh install of grafana/zabbix/plugin. Data is being shown in zabbix, just not in grafana. 
If you have any questions, just let me know thanks.

**Comment by alexanderzobnin** [comment]
Probably the account you're using for connecting to the API has no read access.
